runtime/lua/vim/lsp/semantic_tokens.lua

```diff
@@ -638,6 +647,43 @@ api.nvim_set_decoration_provider(namespace, {
   end,
 })
 
+--- Show the token under the cursor
+---
+---@param winnr (number|nil) Window handle or 0 for current window (default)
+M.show_token_at_cursor = function(winnr)
+  winnr = winnr or 0
+  local cursor = api.nvim_win_get_cursor(winnr)
+
+  local highlights = STHighlighter.active[1].client_state[1].current_result.highlights
```

Comment:
I think `state` should be indexed with the buffer number and it should probably loop over client_state because `client_state[1]` only works if the lsp client with the semantic tokens happens to have the id 1

Reply:
Like so?

Is there a usecase for specifying buffer and/or window?

Reply:
Yes, `opts` is the best way (also for the treesitter PR). I'll make that change!

Reply:
Or `show_token_at_pos`, with empty argument defaulting to cursor? (Similar to my open treesitter PR?)